tooling/build-types-and-manifests.ts

```diff
@@ -59,11 +68,13 @@ addManifest(
       default_title: 'Secure Compose', // eslint-disable-line @typescript-eslint/naming-convention
       default_icon: '/img/logo/flowcrypt-logo-64-64.png', // eslint-disable-line @typescript-eslint/naming-convention
     };
+    delete manifest.minimum_chrome_version;
     (manifest.browser_specific_settings as messenger._manifest.FirefoxSpecificProperties).strict_min_version = '102.0';
     manifest.background = {
       type: 'module',
       scripts: ['/js/service_worker/background.js'],
     };
+    manifest.content_scripts?.splice(0, 1);
```

Comment:
do we need to remove array item here? I thought we use `content_scripts` for injecting js scripts, but extension works well even after this change

Reply:
Hmm, looks like extension works when opened as separate "Encrypted inbox", but doesn't decrypt messages directly in Thunderbird UI, as it's JS files are not injected.

In [thunderbirdContentScriptRegistration](https://github.com/FlowCrypt/flowcrypt-browser/blob/d70bf0f2f8452291ab3c2994d327d7c9ef48830b/extension/js/service_worker/bg-handlers.ts#L156) we extract files from `content_scripts` property, but after this change it contains only `./js/common/oauth2/oauth2_inject.js` and not files needed for processing messages.

Reply:
Thank you for pointing it out @sosnovsky. I made a mistake I guess after this change where everything continue to work on allthroughtout. Retesting it again indeed do not work properly. I revert my changes, close and re-open TB and tested it again.